README.md

```diff
@@ -25,15 +25,25 @@ Our Android template: **[template](https://github.com/nimblehq/android-templates
 
 4. Update `android_version_code` and `android_version_name` in `template/build.gradle`
 
+## Wiki
+
+Learn more about our Android templates on the [Wiki](https://github.com/nimblehq/android-templates/wiki).
+
+## License
+
+This project is Copyright (c) 2014 and onwards. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
 ## About
 
-<a href="https://nimblehq.co/">
-  <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png">
-    <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-160.png">
-  </picture>    
-</a>
```

Comment:
@lydiasama Hmm, why do we remove this block for light mode? I think we should apply this for all places? 🤔

Reply:
@luongvo I copied the ## About and the rest from another template. Shall we use the same with another template of Nimble?

Reply:
@lydiasama I think other Nimble's templates or repositories should take care of migrating to support light/dark mode for the company icon instead. Otherwise, we will drop the effort from @sleepylee https://github.com/nimblehq/android-templates/pull/239 🥲

Reply:
@luongvo Are you sure that block can help? Since I did not update the large Nimble logo at the top of README file but I still see it as the same logo as my update. 😅 
![Screen Shot 2565-09-16 at 11 08 21 AM](https://user-images.githubusercontent.com/12026942/190554845-670d3873-def6-42fe-a1d5-e4e6e57ac872.png)

Reply:
@lydiasama As I said, the code block from @sleepylee is to show the light logo for `light mode`. This is the result after removing in Github's light mode.

![image](https://user-images.githubusercontent.com/16315358/190556539-5076a20a-23da-4a78-aab6-b8397d7f4d8b.png)

Reply:
@luongvo You said light/dark mode 😓 
>@lydiasama I think other Nimble's templates or repositories should take care of migrating to support light/dark mode

So I don't know what you mean by dark mode. 😅

Reply:
@lydiasama I mentioned light mode at first 🥲 https://github.com/nimblehq/android-templates/pull/310#discussion_r972586972

Reply:
@luongvo You should mention only in light mode because it's correct in dark mode already. 🙈